### PR TITLE
Books page integration

### PR DESCRIPTION
**Pull request by mekarpeles**
Followup re: #6277 

### Testing

on http://ol-dev1.us.archive.org:1337/

### Stakeholders

@cdrini @seabelis @mekarpeles @jimchamp @mheiman @bfalling

### CODE REVIEW

**Comment by jimchamp** [comment]
This branch contains the following PRs:
#6322
#6313
#6299
#6286
#6277
#6276
#6266
#6265
#6261

File: openlibrary/templates/type/work/editions_datatable.html [hunk 1]
```diff
@@ -1,8 +1,14 @@
-$def with (work, editions=None, edition=None)
+$def with (work, editions=None, edition=None, editions_limit=None)
 
 $ book_keys = []
 $ edition_list_start = time()
 
+$if editions_limit and len(editions) and len(editions) < work.edition_count:
```
Comment (mekarpeles) [inline openlibrary/templates/type/work/editions_datatable.html:6]:
```suggestion
$if editions_limit or (len(editions) < work.edition_count):
```

File: openlibrary/templates/type/edition/view.html [hunk 2]
```diff
@@ -182,7 +183,9 @@
       <div class="work-line">
         $_("An edition of") <a href="$work.key?edition=$(ocaid)">$work_title$cond(work.subtitle, ': %s' % work.subtitle)</a>
         $if work.first_publish_year:
-          ($work.first_publish_year)
+            <span class="first-published-date">
+              ($_("First published in %s", work.first_publish_year))
```
Comment (jimchamp) [inline openlibrary/templates/type/edition/view.html:187]:
```suggestion
            <span class="first-published-date" title="$_('First published in %s', work.first_publish_year)">
              ($work.first_publish_year)
```

**Comment by jimchamp** [comment]
Committed change to the first publish year:

publish_year_title

File: openlibrary/templates/type/edition/view.html [hunk 3]
```diff
@@ -197,29 +200,80 @@ <h2 class="edition-byline">$_("by") $:authors_byline</h2>
...
-            </h4>
+          <div class="edition-omniline">
+            $if edition.publish_date:
+              <div class="edition-omniline-item">
+                <div>$_("Publish Date")</div>
+                <span itemprop="datePublished">$edition.publish_date</span>
+              </div>
+            $if edition.publishers:
+              <div class="edition-omniline-item">
+                <div>$_("Publishers")</div>
```
Comment (jimchamp) [inline openlibrary/templates/type/edition/view.html:213]:
```suggestion
                <div>$_("Publisher")</div>
```

**Review by cdrini — COMMENTED** [review]
Tested:
- normal work: https://testing.openlibrary.org/works/OL16612125W/Journeys_of_women_in_science_and_engineering
- readable work: https://testing.openlibrary.org/works/OL7000994W/The_little_corner_never_conquered
- Non available work: https://testing.openlibrary.org/works/OL21703472W/Discover_and_Do
- Book url https://testing.openlibrary.org/books/OL33811847M/Discover_and_Do

Bugs for future PRs:
- omnibar sits overtop profile menu
- want to read button sits above profile menu on tablet mode

Code has been reviewed/approved by @mekarpeles and @jimchamp ; please feel free to merge and deploy!

**Comment by cdrini** [comment]
(Letting you merge @jimchamp , since @mekarpeles ' the assignee/reviewer; I just did some light testing :) )

**Pull request merged**